pymeasure/instruments/hp/hp3478A.py

```diff
@@ -37,126 +37,68 @@
 
 # classes for the decoding of the 5-byte status word
 
-
-class Status_bytes(ctypes.Structure):
-    """
-    Support-Class for the 5 status byte of the HP3478A
-    """
-    _fields_ = [
-        ("byte1", c_uint8),
-        ("byte2", c_uint8),
-        ("byte3", c_uint8),
-        ("byte4", c_uint8),
-        ("byte5", c_uint8)
-    ]
+# STATUS_BYTES = 5
```

Comment:
```suggestion
```

Reply:
Sorry that was a lfetover, which I did not clean up properly